src/OpenPr0gramm/FormData/LogInData.cs

```diff
@@ -8,12 +8,18 @@ public class LogInData
         [AliasAs("name")]
         public string Name { get; set; }
         [AliasAs("password")]
-        public string Password { get; set; }
-        public LogInData(string name, string password)
+        public string Password { get; set; }
+        [AliasAs("token")]
+        public string Token { get; set; }
+        [AliasAs("captcha")]
+        public string Captcha { get; set; }
+        public LogInData(string name, string password, string token, string captcha)
```

Comment:
I'd rename `captcha` to `captchaSolution`

Reply:
I've renamed `token` to `captchaToken` as well to prevent confusion.